Graphics/Vulkan/VulkanGraphicsRenderPass.cpp

```diff
@@ -0,0 +1,16 @@
+// Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root
+// for more information.
+
+#include <NovelRT/Graphics/Vulkan/VulkanGraphicsRenderPass.hpp>
+
+namespace NovelRT::Graphics::Vulkan
+{
+    VulkanGraphicsRenderPass::VulkanGraphicsRenderPass(VkRenderPass renderPass) noexcept : _vulkanRenderPass(renderPass)
+    {
+    }
+
+    VkRenderPass VulkanGraphicsRenderPass::GetVulkanRenderPass() const noexcept
+    {
+        return _vulkanRenderPass;
+    }
+}
```

Comment:
GitHub lags and doesn't auto-open the comment section _every time i happen to stumble on an EOL issue_. It's pretty funny actually, lol